apptools/logger/plugin/tests/test_logger_service.py

```diff
@@ -11,9 +11,13 @@
 import unittest
 from unittest import mock
 
-from apptools.logger.plugin.logger_service import LoggerService
+from apptools._testing.optional_dependencies import pyface, requires_pyface
 
+if pyface is not None:
+    from apptools.logger.plugin.logger_service import LoggerService
 
+
+@requires_pyface
```

Comment:
_Side comment:_ I'm kind of surprised that this is the only place we're testing `pyface` functionality, but I guess that's because we're ignoring all the `apptools.undo` module unit tests.

Reply:
> because we're ignoring all the `apptools.undo` module unit tests.

Wait - we have tests for apptools.undo?!

Reply:
Sorry, my mistake - I saw the `apptools.undo` module still exists and didn't realise that it wasn't populated with unit tests.

In which case, this makes complete sense